src/program.rs

```diff
@@ -350,6 +420,41 @@ macro_rules! declare_builtin_function {
     };
 }
 
+/// Tombstone function for when we call an invalid syscall (e.g. a syscall deactivated through a
```

Comment:
"when we call" makes it sound like runtime when it is about verification instead. Maybe "for leaving gaps in a sparse function registry"